src/client.js

```diff
@@ -30,7 +30,9 @@ client
   .on('warn', (warn) => client.logger.warn(warn));
 
 // Login client
-client.login(BOT_TOKEN);
+if(ENV !== 'testing') {
```

Comment:
possibly should add variables in constants but I was thinking 3 vars:
- testing (for unit testing)
- development
- production 

Not sure on diff of last two but possibly something that will be needed in the future.

Reply:
Yeah, I like this